nerf.md

```diff
@@ -0,0 +1,161 @@
+Title: Taichi NeRF (Part 1): Develop and Deploy Instant NGP without writing CUDA
+
+Imagine this: when you flip through a photo album and see pictures of past family trips, do you want to revisit those places and relive those warm moments? When browsing an online museum, do you want to freely adjust your perspective, observe the details of the exhibits up close, and enjoy a full interaction with the cultural relics? When doctors face patients, can they significantly improve diagnostic accuracy and efficiency by synthesizing a 3D perspective of the affected area based on images and providing estimates of lesion size and volume?
+
+NeRF (Neural Radiance Field) technology is the key to making these things a reality. It can reconstruct the 3D representation of a scene from multi-angle captured images and generate images of the scene from any viewpoint and position (new viewpoint synthesis).
+
+The following video demonstrates using NeRF technology to achieve 3D roaming by capturing some static images of Taichi offices with a mobile phone:
```

Comment:
```suggestion
The following video demonstrates using NeRF technology to achieve 3D roaming by capturing some static images of Taichi offices with a mobile phone:
```
The following video demonstrates how NeRF technology is used to achieve 3D roaming by capturing some static images of Taichi offices with a mobile phone: